app/views/assessments/viewFeedback.html.erb

```diff
@@ -6,7 +6,7 @@
 <%= @score.feedback %>
 Score for this problem: <%= @score.score.to_f().round(1) %>
 
-Graded by: <%= if @score.grader && @score.grader.user then @score.grader.user.email end %>
+Graded by: <%= if @score.grader && @score.grader.user then @score.grader.user.email elsif @assessment.has_autograder? then "Autograder" end %>
```

Comment:
There are other places this needs to be resolved as well - for example:

https://github.com/UBAutograding/Autolab/blob/c7cb2382bc6e6993fa82dff1b097bdf5acdc2eee/app/views/scores/_score.html.erb#L10
https://github.com/UBAutograding/Autolab/blob/c7cb2382bc6e6993fa82dff1b097bdf5acdc2eee/app/views/scores/show.html.erb#L10